snippets/article-card.liquid

```diff
@@ -23,17 +23,17 @@
       assign ratio = media_aspect_ratio
     endif
   -%}
-  <div class="card-wrapper underline-links-hover">
+  <div class="article-card-wrapper card-wrapper underline-links-hover">
     {% comment %} <article aria-labelledby="Article-{{ article.id }}"> {% endcomment %}
     <div class="card article-card
-      card--{{ settings.card_style }}
+      card--{{ settings.blog_card_style }}
       {% if media_height and media_height != 'adapt' %} article-card__image--{{ media_height }}{% endif %}
       {% if article.image and show_image %} card--media{% else %} card--text{% endif %}
-      {% if settings.card_style == 'card' %} color-{{ settings.card_color_scheme }} gradient{% endif %}
-      {% if settings.card_style == 'card' and media_height == nil and article.image == empty or show_image == false %} ratio{% endif %}"
+      {% if settings.blog_card_style == 'card' %} color-{{ settings.blog_card_color_scheme }} gradient{% endif %}
+      {% if settings.blog_card_style == 'card' and media_height == nil and article.image == empty or show_image == false %} ratio{% endif %}"
       style="--ratio-percent: {{ 1 | divided_by: ratio | times: 100 }}%;"
     >
-      <div class="card__inner {% if settings.card_style == 'standard' %} color-{{ settings.card_color_scheme }} gradient{% endif %}{% if article.image and show_image or settings.card_style == 'standard' %} ratio{% endif %}" style="--ratio-percent: {{ 1 | divided_by: ratio | times: 100 }}%;">
+      <div class="card__inner {% if settings.blog_card_style == 'standard' %} color-{{ settings.blog_card_color_scheme }} gradient{% endif %}{% if article.image and show_image or settings.blog_card_style == 'standard' %} ratio{% endif %}" style="--ratio-percent: {{ 1 | divided_by: ratio | times: 100 }}%;">
         {%- if show_image == true and article.image -%}
           <div class="article-card__image-wrapper card__media">
             <div class="article-card__image media media--hover-effect" {% if section.settings.media_height == 'adapt' %} style="padding-bottom: {{ 1 | divided_by: article.image.aspect_ratio | times: 100 }}%;"{% endif %}>
```

Comment:
> We'll need to consider associating a theme setting to apply to [Pages for the search results page](https://screenshot.click/16-41-g8q0p-l09jf.png). My recommendation would be to follow Blog posts for now. I know we need to improve those cards #1509 but that can be done later. We would at least give them a home for better grid results.

My interpretation of @melissaperreault's recommendation is to associate Pages within search results with the Blog cards settings until we have new theme settings specifically for Pages. This way, both Pages and Blogs within the search results will have better consistency.

Right now, Pages within the search results are using the Product cards settings:

<img src="https://user-images.githubusercontent.com/28404165/174667089-b9dbb4a4-345c-418e-87b6-946f469b8ef4.png" width="500" height="auto">

Reply:
I think that makes sense 🤔 I will let @duygukalaycioglu confirm but also this isnt in scope of this PR and I can address it in a follow up if that is ok 👍